### PR TITLE
Feat/footer

### PR DESCRIPTION
**Pull request by Kenny-Gin1**

### CODE REVIEW

File: apps/web/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        >
+          <TwitterIcon size={40} round />
+        </a>
+        <a
+          href="https://example.com"
+          target="_blank"
+          rel="noreferrer"
+        >
+          <LinkedinIcon size={40} round />
+        </a>
```
Comment (ericzakariasson) [inline apps/web/src/components/Footer.tsx:25]: will we have a linkedin? i guess we can remove it for now?

File: apps/web/src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <a
+          href="https://example.com"
```
Comment (ericzakariasson) [inline apps/web/src/components/Footer.tsx:27]: need to update this!

File: apps/web/src/pages/_app.tsx [hunk 3]
```diff
@@ -28,6 +29,7 @@ function MyApp({ Component, pageProps }: AppProps) {
               <main className="m-auto max-w-3xl px-4 py-8">
                 <Component {...pageProps} />
               </main>
+              <Footer className="absolute bottom-0 right-[calc(50%_-_16rem)] mb-3"/>
```
Comment (ericzakariasson) [inline apps/web/src/pages/_app.tsx:32]: i wonder if we should have it a bit more low-key in the sidebar instead like we have it in the task? Then it won't be so distracting from the editing experience

I've updated the design in Figma, it's like this now
image

**Review by ericzakariasson — COMMENTED** [review]
nice! some thoughts!

File: apps/web/src/pages/_app.tsx (hunk 3 above)
Comment (Kenny-Gin1) [reply apps/web/src/pages/_app.tsx:32]: Makes sense to be more lowkey for sure!

**Pull request merged**